src/packs/npcs/LOG

```diff
@@ -1 +1,5 @@
-2023/08/03-12:26:23.237511 7f84d67fc700 Delete type=3 #1
+2023/10/31-19:47:07.642225 7f112eefd700 Recovering log #3
```

Comment:
todos estos ficheros que estan en npcs son necesarios? @Eryx5502

Reply:
Voy a echarle un ojo, porque la verdad es que no estoy seguro...

Reply:
Creo que todos estos nuevos archivos son debido a cómo funcionan y se generan compendios de npcs en la v11, a Julen y a mi nos pasó lo mismo con un módulo de npcs al realizar la migración

Reply:
He estado mirando y no he encontrado una respuesta clara sobre sí los archivos LOG son o no necesarios para el correcto funcionamiento de la base de datos que usa foundry. Lo que sí he visto es que recomiendan no subir los archivos de la base de datos, sino usar el CLI de foundry para mantener los compendios "compilándolos desde json". He creado el issue #186 al respecto!

Creéis que es mala idea dejarlo así de momento hasta que cambiemos a ese modelo, una vez que #186 esté implementado?

Reply:
Diria que si son logs los podemos borrar

Reply:
Parece que no importa si los borramos, aunque antes he visto un comportamiento inestable (de repente, al crear nuevos mundos con el sistema sin logs algunos de los packs me salían vacíos). Pero no he podido reproducirlo después de borrar la carpeta del sistema y los mundos y crear de cero.